packages/gatsby/src/utils/page-data-helpers.ts

```diff
@@ -0,0 +1,50 @@
+import { IGatsbyPage } from "../redux/types"
```

Comment:
Extracted to separate module from `packages/gatsby/src/utils/page-data.ts` + addition of `getPagePathFromPageDataPath` to reverse `/page-data/<page-path>/page-data.json` to `<page-path>`